### PR TITLE
CI: add aws.sh image builder test in CIV ci pipeline

### PR DESCRIPTION
**Pull request by jabia99**
CIV tool is being executed in aws.sh (IB CI), but it's running a specific release,
so there's a lot of manual work to keep this release updated. If we run
this test in out own CI and publish the container image, IB can always
get the last version of CIV without worrying if it's going to cause
errors.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ stages:
     - sudo dnf update -y && sudo dnf install -y podman
   variables:
     RUNNER: aws/fedora-35-x86_64
+    INTERNAL_NETWORK: ["true"]
```
Comment (narmaku) [inline .gitlab-ci.yml:13]: Q: What is this env var used for?

Comment (jabia99) [reply .gitlab-ci.yml]: It allows the runners to be in the VPN

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -62,14 +64,47 @@ azure:
...
 
+.rhel_runners: &rhel_runners
+    RUNNER:
+      - aws/centos-stream-8-x86_64
+      - aws/rhel-8.4-ga-x86_64
+      - aws/rhel-8.6-ga-x86_64
+      - aws/rhel-9.0-ga-x86_64
+      - aws/rhel-8.7-nightly-x86_64
+      - aws/rhel-9.1-nightly-x86_64
+      - aws/centos-stream-9-x86_64
```
Comment (narmaku) [inline .gitlab-ci.yml:83]: I thick centos should be in .centos_runners

Comment (jabia99) [reply .gitlab-ci.yml:83]: centos is included in this category as it is considered a rhel like system, unlike fedora. I think the only reason this rhel-fedora separation is made in the yaml is to give acces to rhel like systems to the internal network. But we can separate centos

**Pull request merged**